### PR TITLE
Added fully featured v16 config as an example

### PR DESCRIPTION
**Pull request by Pietfried**
## Describe your changes
Tested and works with SteVe

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] If OCPP 2.0.1: I have updated the [OCPP 2.0.1 status document](https://github.com/EVerest/libocpp/tree/main/doc/ocpp_201_status.md)
- [x] I read the [contribution documentation](https://github.com/EVerest/EVerest/blob/main/CONTRIBUTING.md) and made sure that my changes meet its requirements

### CODE REVIEW

File: config/v16/config-full.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        "ConnectorEvseIds": "1,2",
+        "AllowChargingProfileWithoutStartSchedule": false,
+        "WaitForStopTransactionsOnResetTimeout": 60,
+        "QueueAllMessages": true,
+        "MessageTypesDiscardForQueueing": "Heartbeat",
+        "MessageQueueSizeThreshold": 5000,
+        "SupportedMeasurands": "Energy.Active.Import.Register,Energy.Active.Export.Register,Power.Active.Import,Voltage,Current.Import,Frequency,Current.Offered,Power.Offered,SoC",
+        "MaxMessageSize": 65000,
+        "TLSKeylogFile": "/tmp/ocpp_tls_keylog.txt",
+        "EnableTLSKeylog": true
```
Comment (hikinggrass) [inline config/v16/config-full.json:64]: I think we should set this to false by default

Comment (Pietfried) [inline config/v16/config-full.json:64]:
```suggestion
        "EnableTLSKeylog": false
```

File: config/v16/config-full.json [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            "ChargePointMaxProfile",
+            "TxDefaultProfile",
+            "TxProfile"
+        ],
+        "MaxCompositeScheduleDuration": 31536000,
+        "WebsocketPingPayload": "Hello from EVerest!",
+        "WebsocketPongTimeout": 5,
+        "UseSslDefaultVerifyPaths": true,
+        "VerifyCsmsCommonName": true,
+        "VerifyCsmsAllowWildcards": false,
```
Comment (Pietfried) [inline config/v16/config-full.json:50]:
```suggestion
        "VerifyCsmsAllowWildcards": true,
```

File: config/v16/config-full.json [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        },
+        "TimeOffset": "02:00",
+        "NextTimeOffsetTransitionDateTime": "2024-01-01T00:00:00",
+        "TimeOffsetNextTransition": "01:00",
+        "CustomIdleFeeAfterStop": false,
+        "MultiLanguageSupportedLanguages": "en, nl, de, nb_NO",
+        "CustomMultiLanguageMessages": true,
+        "Language": "en"
+    }
+}
```
Comment (Pietfried) [inline config/v16/config-full.json:180]:
```suggestion
}

```

**Pull request merged**